Sources/SwiftParser/Modifiers.swift

```diff
@@ -173,4 +155,44 @@ extension Parser {
       arena: self.arena
     )
   }
+
+  mutating func parseAccessLevelModifier() -> RawDeclModifierSyntax {
+    let (unexpectedBeforeName, name) = expectAny(
+      [.privateKeyword, .fileprivateKeyword, .internalKeyword, .publicKeyword],
+      default: .privateKeyword
```

Comment:
It doesn’t really matter too much but I think `.internalKeyword` would be the better default here because it’ Swift’s default access level if you don’t specify anything.

Reply:
Ah thats right. I fixed it.